app/models/suggestions_job_status.rb

```diff
@@ -0,0 +1,2 @@
+class SuggestionsJobStatus < ApplicationRecord
```

Comment:
`done` appears to be a required variable, does it make sense to have a `presence` validation?